.gitignore

```diff
@@ -5,6 +5,8 @@ dist
 /dist-newstyle/
 *.o
 *.hi
+.ghc.environment.*
+*.swp
```

Comment:
This one however is a a generic temp file; and since we didn't have `*~` either, it'd suggest this gitignore file follows the idiom of gitignoring editor-related temp-files in the per-user Git configuration...